test/uk/gov/hmrc/helptosavefrontend/controllers/AuthSupport.scala

```diff
@@ -66,11 +66,11 @@ trait AuthSupport extends TestSupport {
   val countryCode = "GB"
   val itmpAddress = ItmpAddress(Some(line1), Some(line2), Some(line3), None, None, Some(postCode), Some(countryCode), Some(countryCode))
 
-  val mockedRetrievals = new ~(name, email) and dob and itmpName and itmpDob and itmpAddress and Enrolments(Set(enrolment))
+  val mockedRetrievals = new ~(name, email) and Option(dob) and itmpName and itmpDob and itmpAddress and Enrolments(Set(enrolment))
   //val mockedRetrievals = new ~(new ~(new ~(new ~(new ~(new ~(name, email), dob), itmpName), itmpDob), itmpAddress), Enrolments(Set(enrolment)))
-  val mockedMissingUserInfo = new ~(new ~(new ~(new ~(new ~(new ~(name, noEmail), dob), itmpName), itmpDob),
+  val mockedMissingUserInfo = new ~(new ~(new ~(new ~(new ~(new ~(name, noEmail), Option(dob)), itmpName), itmpDob),
```

Comment:
could we not reuse the `and` from the implicit class instead of doing `new~(new~(new~(...`?

Reply:
I changed it myself